arionBackend/tests/api/test_hierarchy.py

```diff
@@ -39,6 +46,15 @@ def test_get_hierarchy_by_id_invalid_id(self):
         request = self.client.get('/api/hierarchy/esper/0', follow=True)
         self.assertEqual(request.status_code, status.HTTP_400_BAD_REQUEST)
 
+    def test_get_hierarchy_by_id_not_found(self):
+        """
+        Tests if the hierarchy cannot be retrieved, if there is no hierarchy with the id.
+        :return: 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3AS1131)